crates/bins/src/bin/datadog-static-analyzer.rs

```diff
@@ -128,6 +131,26 @@ fn main() -> Result<()> {
         "print-violations",
         "print a list with all the violations that were found",
     );
+    opts.optflag(
+        "",
+        "fail-on-any-violation-error",
+        "exit a non-zero return code if there is one violation with the error severity",
+    );
+    opts.optflag(
```

Comment:
Normally, this kind of thing is an enum flag, like

`--fail-on-violation=error`

I think it would be easier and more obvious for the user if there were only one flag that works like that.

Reply:
yes, I was thinking about this and I am changing this. Will follow up with another PR